test_scripts/API/Restructuring_OnResetTimeout/013_BC_OnResetTimeout_to_DialNumber.lua

```diff
@@ -0,0 +1,61 @@
+------------------------------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+------------------------------------------------------------------------------------------------------------------------
+-- Description: Check SDL does not reset timeout for Mobile app response for a specific RPC in case
+--  HMI sends 'OnResetTimeout(resetPeriod)' notification
+-- Applicable RPCs: 'DialNumber', 'Alert' (with soft buttons), 'SubtleAlert' (with soft buttons)
+------------------------------------------------------------------------------------------------------------------------
+-- Preconditions:
+-- 1) Default SDL timeout is 10s (defined in .INI by 'DefaultTimeout' parameter)
+--
+-- In case:
+-- 1) App sends applicable RPC
+-- 2) SDL transfers this request to HMI
+-- 3) HMI sends 'BC.OnResetTimeout' notification to SDL right after receiving request with 'resetPeriod = 12s'
+-- 4) HMI sends response after delay of 14s
+-- SDL does:
+--  - not apply default timeout
```

Comment:
Does this mean that _onResetTimeout_ is not applied to DialNumber/Alert with SB etc? Is so, why is it being hardcoded for these RPCs? Can you please point to the specification for that?

Reply:
@atiwari9 Yes this is correct. There is an old specific requirement for `Alert` (with soft buttons) and for `DialNumber` RPCs:
>  SDL must not apply "default timeout for RPCs processing" (that is, SDL must always wait for HMI response as long as it takes and not return GENERIC_ERROR to mobile app)

Reply:
Where is this quoted, Can you please add a link to that as well in the scripts where applicable for each of these RPCs. Thanks.